cpp/include/cudf/detail/utilities/hash_functions.cuh

```diff
@@ -90,11 +191,6 @@ struct MurmurHash3_32 {
   MurmurHash3_32() = default;
   constexpr MurmurHash3_32(uint32_t seed) : m_seed(seed) {}
 
-  [[nodiscard]] __device__ inline uint32_t rotl32(uint32_t x, uint32_t r) const
-  {
-    return __funnelshift_l(x, x, r);  // Equivalent to (x << r) | (x >> (32 - r))
```

Comment:
We no longer need to define `rotl32` in `MurmurHash3_32` because it's identical to the `cudf::detail::rotate_bits_left` utility that I moved into this file.